src/components/NewItemButton/NewItemButton.styles.ts

```diff
@@ -0,0 +1,59 @@
+import { NewItemButtonState } from './NewItemButtonState.type';
+
+import type { BaseProps } from '@/types/BaseProps';
+
+export type NewItemButtonConfig = {
+  state?: Partial<Record<NewItemButtonState, BaseProps>>;
+  innerElements?: {
+    text?: BaseProps;
+  };
+} & BaseProps;
+
+export const defaultConfig = {
+  display: 'inline-flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  gap: '$space-component-gap-small',
+  h: '120px',
+  w: 'fit-content',
+  minWidth: '120px',
+  borderWidth: '$border-width-small',
+  borderStyle: 'dashed',
```

Comment:
`$border-style-dashed`

Reply:
The `$border-style-dashed` is actually `dashed solid` style, which don't align with the design. Should the key be changed to `dashed`, or is this deliberately made that way?

Reply:
@golas-m  you can change it here, it will be changeged also with #139 PR.